ui-tests/src/test/java/org/openmrs/reference/AdmitExitInpatientTest.java

```diff
@@ -12,7 +13,7 @@
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
-
+@Ignore
```

Comment:
This will be reverted when `AdmitExitInpatientTest` has been excluded from the ci build.